var/spack/repos/builtin/packages/libdc1394/package.py

```diff
@@ -10,17 +10,30 @@ class Libdc1394(AutotoolsPackage):
     """Library providing an API for IEEE 1394 cameras."""
 
     homepage = "https://damien.douxchamps.net/ieee1394/libdc1394/"
-    url      = "https://downloads.sourceforge.net/project/libdc1394/libdc1394-2/2.2.6/libdc1394-2.2.6.tar.gz"
+    url = "https://downloads.sourceforge.net/project/libdc1394/libdc1394-2/2.2.6/libdc1394-2.2.6.tar.gz"
 
-    maintainers = ['traversaro']
+    maintainers = ["traversaro"]
```

Comment:
@traversaro can you review these dependency changes?

Reply:
Ok! Sorry for the late reply.